components/list/list-item-content.js

```diff
@@ -12,13 +12,13 @@ class ListItemContent extends LitElement {
 
 	static get styles() {
 		return [ bodySmallStyles, bodyCompactStyles, css`
-			:host {
```

Comment:
I think these overflow values were meant to handle crazy long string cases as described in [Maya's PR](https://github.com/BrightspaceUI/core/pull/752/files). I think there might also have been some original code to render ellipsis, but if there was, that is no longer the case.

As described in [these Mozilla notes](https://developer.mozilla.org/en-US/docs/Web/CSS/word-break), the `word-break: normal + overflow-wrap: anywhere` achieves essentially the same as `word-break: break-word` (where `break-word` usage on `word-break` also works but is deprecated). Also, `overflow-wrap` is the standardized property that replaced the non-standard `word-wrap` - they map to the exact same thing.